Changes

```diff
@@ -314,6 +314,9 @@ Working version
 - GPR#1970: fix order of floatting documentation comments in classes
   (Hugo Heuzard, review by Nicolás Ojeda Bär)
 
+- GPR#1973: fix compilation of catches with multiple handlers
+  (Vincent Laviron)
```

Comment:
This should move to the trunk section of the changelog

Reply:
Fixed